src/NUnitFramework/framework/nunit.framework.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <AssemblyName>nunit.framework</AssemblyName>
+    <RootNamespace>NUnit.Framework</RootNamespace>
+    <TargetFrameworks>net20;net35;net40;net45;netstandard1.6</TargetFrameworks>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OutputPath>..\..\..\bin\$(Configuration)\</OutputPath>
+    <DocumentationFile>..\..\..\bin\$(Configuration)\$(TargetFramework)\nunit.framework.xml</DocumentationFile>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <DebugType Condition="'$(TargetFramework)' != '' AND '$(TargetFramework)' != 'netstandard1.6'">Full</DebugType>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net20' OR '$(TargetFramework)' == 'net35'">
+    <DefineConstants>TRACE;NUNIT_FRAMEWORK;PARALLEL</DefineConstants>
```

Comment:
One for the clean-up, @jnm2 - I think we could probably remove the `NUNIT_FRAMEWORK` constance? I think that's left from when framework and engine we in the same repo?

Reply:
@ChrisMaddock Also on my list, but in this case, there are files linked into the NUnit and NUnitLite projects so I should double check that none of our code uses these constants.